gpii/node_modules/matchMakerFramework/src/MatchMakerCapabilities.json

```diff
@@ -1,49 +1,38 @@
 [
     {
-        "matchMakerId": "flat",
-        "conditions": {
-            "supported": false
+        "matchMakerId": "Statistical1",
```

Comment:
These changes will break all the other scenarios since they remove the RBMM... I think a good solution would be to have all the possible MMs listed in this capabilities file, and then have the dispatcher filter the content of this file based on the list of MMs found in the config (fluid.filter works well for something like this) - this list should be available as a property of the flowmanager